src/util/logger-utils.ts

```diff
@@ -0,0 +1,82 @@
+import { Writable } from "stream";
+import safeJsonStringify from "safe-json-stringify";
+import bformat from "bunyan-format";
+import { DEBUG } from "bunyan";
+import { createHashWithSharedSecret } from "utils/encryption";
+
+const SENSITIVE_DATA_FIELDS = ["jiraHost", "orgName", "repoName", "userGroup", "userGroup", "aaid", "username"];
```

Comment:
As we find more fields that need to be hashed we can just add/remove from this array